homework01/rudolf/vector.cpp

```diff
@@ -0,0 +1,46 @@
+#include "vector.h"
+
+Vector::Vector()
+{
+	mX=0;
```

Comment:
As discussed Thursday, it's good practice to use initializer list. About the style, our (Optiver) style is `mX = 0;`